tests/test_storageclass_deletion.py

```diff
@@ -0,0 +1,140 @@
+"""
+Automates te following test:-
+https://polarion.engineering.redhat.com/polarion/#/project/OpenShiftContainerStorage/workitem?id=OCS-297
+
+1) Create a Storage Class
+2) Create a PVC
+3) Delete the corresponding storage class
+"""
+import logging
+import yaml
+import os
+from utility.utils import run_cmd
+from ocs import ocp
+from openshift.dynamic import DynamicClient
+from utility import utils, templating
+from kubernetes import config
+from ocsci.enums import StatusOfTest
+
+k8s_client = config.new_client_from_config()
+dyn_client = DynamicClient(k8s_client)
+log = logging.getLogger(__name__)
+
+PVC_YAML = os.path.join(
+    "templates/ocs-deployment", "PersistentVolumeClaim.yaml"
+)
+SC_YAML = os.path.join(
+    "templates/ocs-deployment", "StorageClass.yaml"
+)
+TEMP_PVC_YAML_FILE = '/tmp/pvc_test.yaml'
+TEMP_POD_YAML_FILE = '/tmp/pod_test.yaml'
+TEMP_SC_YAML_FILE = '/tmp/sc_test.yaml'
+
+
+PVC = ocp.OCP(kind='PersistentVolumeClaim')
+PV = ocp.OCP(kind='PersistentVolume')
+SC = ocp.OCP(kind='StorageClass')
+NAMESPACE = ocp.OCP(kind='Project')
+
+
+def create_namespace(**kwargs):
+    '''
+    Creating a project
+    '''
+    project_name = (kwargs['project_name'])
+    project_get = NAMESPACE.get()
+    namespaces = []
+    for i in range(len(project_get['items'])):
+        namespaces.append(project_get['items'][i]['metadata']['name'])
```

Comment:
Start to work with list comprehensions (google it):
namespaces = [item['metadata']['name'] for item in project_get['items']]